src/main/scala/com/microsoft/ml/spark/lightgbm/params/FObjTrait.scala

```diff
@@ -0,0 +1,17 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in project root for information.
+
+package com.microsoft.ml.spark.lightgbm.params
+
+import com.microsoft.ml.spark.lightgbm.dataset.LightGBMDataset
+
+trait FObjTrait extends Serializable {
```

Comment:
Perhaps we should rename this to CustomObjective

Reply:
I think that's what it is called in XGBoost actually, but in lightgbm the parameter is called fobj, so I wanted to keep in line with the terminology in lightgbm

Reply:
SG